helmcharts/notes/templates/_helpers.tpl

```diff
@@ -0,0 +1,67 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "notes.name" -}}
+{{- default .Chart.Name .Values.app.nameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
Can we indent these blocks (and the rest of this file)?

Reply:
This file with the exception of the last two definitions:
notes.requiredSecretName
notes-mysql.secretName

was just generated by helm create.

I believe indenting would add tabs to the places it is used and is not typically done in the stable charts which I tried to mirror as much as possible.  Here is an example:
https://github.com/helm/charts/blob/master/stable/ambassador/templates/_helpers.tpl

Does that change your opinion here?

Reply:
Indenting is typically added to the template via piping to '| indent 6' for example

Reply:
If this is standard, that is fine.  I don't have a strong preference, especially where it is easy to reason about the blocks.